docs/src/main/sphinx/admin/properties-query-management.rst

```diff
@@ -132,6 +132,14 @@ After this period the coordinator will make its best effort to stop the
 query. Note that some operations in planning phase are not easily cancellable
 and may not terminate immediately.
 
+``query.reported-rule-stats-limit``
```

Comment:
I would skip it. Query JSON is internal engine object, no need to expose details of it